Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.15 as builder
+LABEL maintainers="https://tinkerbell.org/community/slack/"
+
+WORKDIR /code
+COPY go.mod go.sum /code/
+RUN go mod download
+
+COPY . /code
+RUN make build
+
+FROM scratch
+ARG DEFAULT_PORT=9090
+ENV PBNJ_PORT=${DEFAULT_PORT}
```

Comment:
Why would we want an arg for default port? Seems a little weird to me. Changing the port can be done at run time via env and the arg isn't really used in the build process so doesn't really need to change there.

Reply:
sure, I'll remove this.